src/Samples/SubModel.Views/SubModel.Views.csproj

```diff
@@ -26,6 +26,12 @@
     <Reference Include="WindowsBase" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Please revert this file. VS does this automatically, but it's not needed. (.NET Core 3.0 will bring and end to the csproj workarounds.)

Reply:
Oh, shoot.  I thought I had.  Nice catch.